scaleapi/nucleus-python-client
# Creating specific datasets for scenes

# Summary
We'll treat datasets that contain scenes differently in the frontend (additional tab) compared to those without scenes. Therefore we need to introduce an attribute for a dataset which specifies the type of dataset as a boolean (`is_scene`). This attribute needs to be set during dataset creation. 

Regarding the minor version, I assumed, we need to see if we merge [Gunnar's PR](https://github.com/scaleapi/nucleus-python-client/pull/159) first. So if this one would be merged first, it should be `0.5.0` instead. 

# How to test
This PR depends on the backend changes introduced in PR [#33106](https://github.com/scaleapi/scaleapi/pull/32750). To test it, do the following
- set `NUCLEUS_ENDPOINT = "http://localhost:3000/v1/nucleus"` in `nucleus/constants.py`
- Install this client from `pip install .` from source folder
- Check out `scaleapi` branch `mark-add-scene-type`
- yarn `dev-prodclone-backend`
- yarn `dev-prodclone-sfn`
- Create a dataset and upload a scene through the API. I can share a little script for that, let me know if you need sth.
